list

```diff
@@ -676,6 +690,7 @@ kutt.it
 l-i-nk.me
 l.linklyhq.com
 l.prageru.com
+l1nq.com
```

Comment:
It's interesting that this one looks more like a redirector that redirect anything to  https://www.encurtador.dev/redirecionamento/?

Reply:
It 301 redirects to a redirection page at ```https://www.encurtador.dev/redirecionamento/{LinkID}``` and does some sort of checks, then redirects to target.
Here's an example without a redirection page (still goes through encurtador.dev but instant 301): https://l1nq.com/ti1aa Archive: https://archive.is/PQwWk